server/py/services/api/crud/alert_activation.py

```diff
@@ -69,6 +71,7 @@ def _prepare_notifications_states(
             # count successes and collect unique errors for failures
             if reason:
                 notification_errors[kind]["errors"].add(reason)
+                notification_errors[kind]["failed_count"] += 1
```

Comment:
If it is the only place where you increments the `failed_count`, why should we have it? it should be the length of the `errors`.

Reply:
@roei3000b no, it is not because errors is set, which means it has **unique** errors only